### DISCUSSION TITLE
remove device or uninstall all

### DISCUSSION THREAD

**Discussion by kohj78** [T-Troll/alienfx-tools#199] [Q&A]
I have have loaded wrong device (G15) into light tab and can find no way to remove it. I have M15 R7 and want to restart from begining and cant figure out how to uninstall. I have deleted all files but when i open GUI its is the same with G15 selected still.  Also thank you for makeing this it's fantastic.

**Comment by T-Troll** [comment]
Thank you.

I see your device not in database (no one share mappings for it yet), so you need to configure it out manually. BTW, what do you have into detection list (sometimes different models uses the same devices and maps)?
So, just open `regedit` and delete entry `HKCU/SOFTWARE/Alienfx_DSK` - this will clear all lights, grids and zones.

I don't clean it at uninstall because Windows installer quite stupid and will clear it every app update (it's even worse).

**Comment by kohj78** [comment]
Thank you, in device list i have M15 R1, Aurora R12 and G5 5515. I can send mapping one i get it figured out.

**Comment by T-Troll** [comment]
Aha, common 550 chip (it's Tron, but who know you mappings...)
Keyboard is something new, so yes, manual set.
